### PR TITLE
Enable pnpm in tools

### PR DESCRIPTION
**Pull request by pat-trunk-io**

### CODE REVIEW

**Comment by pat-trunk-io** [comment]
Current dependencies on/for this PR:
* main
  * **PR #308** <a href="https://app.graphite.dev/github/pr/trunk-io/plugins/308" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/trunk-io/plugins/308?utm_source=stack-comment).

File: tools/pnpm/plugin.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        - os:
+            linux: linuxstatic
+            macos: macos
+          cpu:
+            x86_64: x64
+            arm_64: arm64
+          url: https://github.com/pnpm/pnpm/releases/download/${version}/pnpm-${os}-${cpu}
+  definitions:
+    - name: pnpm
+      known_good_version: v8.6.1
```
Comment (TylerJang27) [inline tools/pnpm/plugin.yaml:16]: Can you omit the v here and add it to the download url?

File: tools/pnpm/pnpm.test.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { makeToolTestConfig, toolTest } from "tests";
+
+toolTest({
+  toolName: "pnpm",
+  toolVersion: "v8.6.1",
```
Comment (TylerJang27) [inline tools/pnpm/pnpm.test.ts:5]: Also omit the v here?

File: tools/pnpm/plugin.yaml (hunk 1 above)
Comment (pat-trunk-io) [reply tools/pnpm/plugin.yaml]: Addressed

File: tools/pnpm/pnpm.test.ts (hunk 2 above)
Comment (pat-trunk-io) [reply tools/pnpm/pnpm.test.ts]: Addressed

File: tools/pnpm/plugin.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (pat-trunk-io) [reply tools/pnpm/plugin.yaml]: Addressed

**Review by TylerJang27 — APPROVED** [review]

**Pull request merged**